### PR TITLE
Added sample of using VMware Storage Policy API

### PR DESCRIPTION
**Pull request by verdel**
I would like to add an example implementation of:
 - Getting information about VM Home and Virtual Disk storage policies
 - Assignment storage policy to the VM Home and Virtual Disks

using the VMware Storage Policy API.

### CODE REVIEW

**Comment by verdel** [comment]
Hi, @pgbidkar can you please review the PR?

**Comment by pgbidkar** [comment]
> Hi, @pgbidkar can you please review the PR?

Hi @verdel , I am extremely sorry. I am not sure how I missed this PR. Please give me a day time I will review it

Thanks for the patience 
Pavan

**Comment by verdel** [comment]
> Hi @verdel , I am extremely sorry. I am not sure how I missed this PR. Please give me a day time I will review it

No problem. I will wait. Thank you for your time.

File: samples/get_vm_storage_policy.py [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+            if (vm.name == name):
+                obj.append(vm)
+                return obj
+        else:
+            if re.match(".*{}.*".format(name), vm.name):
+                obj.append(vm)
+    return obj
+
+
+def GetArgs():
```
Comment (pgbidkar) [inline samples/get_vm_storage_policy.py]: Please move this to top of the file to be in consistent with other  sample files

File: samples/get_vm_storage_policy.py [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+    """Supports the command-line arguments listed below.
+    """
+    parser = argparse.ArgumentParser(
+        description='Show VM Home and Virtual Disk Storage Policies')
+    parser.add_argument('-s', '--host',
```
Comment (pgbidkar) [inline samples/get_vm_storage_policy.py]: Use library argument builder

File: samples/get_vm_storage_policy.py (hunk 1 above)
Comment (verdel) [reply samples/get_vm_storage_policy.py]: Changes have also been made in samples/set_vm_storage_policy.py

File: samples/get_vm_storage_policy.py (hunk 2 above)
Comment (verdel) [reply samples/get_vm_storage_policy.py]: Changes have also been made in samples/set_vm_storage_policy.py

**Comment by verdel** [comment]
I changed the scripts a little bit more. I added the option --set_vm_home in samples/set_vm_storage_policy.py. Setting the storage policy for VM Home is now optional.

**Comment by pgbidkar** [comment]
@kumahesh , can you review this sample? Also please verify the sample

**Comment by pgbidkar** [comment]
Hi @verdel , Please can you provide your sample test results? Screen capture of command prompt where Sample is run successfully

**Comment by verdel** [comment]
> Hi @verdel , Please can you provide your sample test results? Screen capture of command prompt where Sample is run successfully

Sure, no problem.
[asciicast](https://asciinema.org/a/261278)

**Comment by kumahesh** [comment]
Works as expected. It is really valuable contribution. It will be merged soon.

**Comment by pgbidkar** [comment]
Hi @verdel , Thanks for the contribution. Merging the PR

**Pull request merged**